UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Services/SubscriptionServiceTest.cs

```diff
@@ -18,9 +20,13 @@ public class SubscriptionServiceTest
     {
         private ID365PayloadValidator _fakeD365PayloadValidator;
         private IOptions<D365PayloadKeyConfiguration> _fakeD365PayloadKeyConfiguration;
-        private D365Payload _fakeD365PayloadDetails;
-        private SubscriptionRequest _fakeSubscriptionRequest;
-        private SubscriptionService _subscriptionService;
+        private D365Payload _d365PayloadDetails;
+        private SubscriptionRequest _subscriptionRequest;        
+        private NotificationType _notificationType;
+        private ISubscriptionService subscriptionService;
```

Comment:
Naming please:
```suggestion
        private ISubscriptionService _subscriptionService;
```

Reply:
that's done while merging min to this branch